docs/PreReleaseNotes.txt

```diff
@@ -31,6 +31,8 @@ Prerelease Notes
   **Commit: 0f79a38
   **[XrdMacaroons] Fix authentication when different tokens are used in the same TCP session**
   **Commit: 4410d56
+  **[XrdHttp] 500 Internal Server Error on unknown checksum algorithm**
+  **Commit: 8c9fbb3
```

Comment:
Out of curiosity, how do you know the final commit hash?

Reply:
That's the one from my repo, I did not know whether I should have left it blank or if I should have put mine as a placeholder... I can remove it if you want